capplets/time-admin/src/time-zone.c

```diff
@@ -437,15 +437,21 @@ void SetupTimezoneDialog(TimeAdmin *ta)
                                               NULL);
     gtk_window_set_default_size (GTK_WINDOW (ta->dialog), 730, 520);
 
-    ta->TZclose =   DialogAddButtonWithIconName(GTK_DIALOG(ta->dialog),
-                                              _("Close"),
-                                              "window-close",
-                                               GTK_RESPONSE_CANCEL);
-
-    ta->TZconfire = DialogAddButtonWithIconName(GTK_DIALOG(ta->dialog),
-                                              _("Confirm"),
-                                              "emblem-default",
-                                               GTK_RESPONSE_OK);
+
+    ta->TZclose = gtk_button_new_with_mnemonic (_("_Close"));
+    gtk_button_set_image (GTK_BUTTON (ta->TZclose), gtk_image_new_from_icon_name ("window-close", GTK_ICON_SIZE_BUTTON));
+    gtk_button_set_use_underline (GTK_BUTTON (ta->TZclose), TRUE);
+    gtk_style_context_add_class (gtk_widget_get_style_context (ta->TZclose), "text-button");
+    gtk_widget_set_can_default (ta->TZclose, TRUE);
+    gtk_dialog_add_action_widget (GTK_DIALOG (ta->dialog), ta->TZclose, GTK_RESPONSE_CANCEL);
+
+    ta->TZconfire = gtk_button_new_with_mnemonic (_("Con_firm"));
```

Comment:
It should be ("_Confirm")

Reply:
You can't use the same the same key accelerator twice.